fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerBuildGroupedCountIndexTest.java

```diff
@@ -76,7 +76,9 @@ private static Stream<String> sourceIndexNames() {
 
     static Stream<Arguments> sourceIndexes() {
         return sourceIndexNames()
-                .map(Arguments::of);
+                .flatMap(sourceIndexName ->
+                        OnlineIndexerBuildIndexTest.randomSeeds().map(seed ->
+                                Arguments.of(sourceIndexName, seed)));
```

Comment:
Would this function be more readable if named `randomSourceIndexesAndSeeds`? 
(following `randomSeeds`'s presedent)

Reply:
I went with `souceIndexesAndRandomSeeds` given that (at least for now) the "random" only applies to the seed. We could definitely consider something like a "random arg", where we pick a random source index and seed. It's possible we could simplify these tests a bit and instead of having one test for each configuration, have one test that supplies a random configuration. But for now, all I did was make the seeds random.